src/ini/ini_read.h

```diff
@@ -19,6 +19,10 @@ typedef int (*ini_callback)(char *section, char *key, char *value, void *data_pt
  * pointed to by data_ptr in a meaningful way. the callback function
  * should return INI_FAIL if there was a fatal problem, or INI_OK otherwise.
  *
+ * Note: The callback function is also called when a new section header is
+ *       encountered with an empty key and value to allow for handling settings
```

Comment:
A bit unclear because of the order, I think it is better to say
"The callback function is also called with an empty key and value when a new section header is encountered, such that the callback can handle settings specified in separate blocks."

Reply:
I reworded this comment as per your suggestion.